applications/protocols_serialization/README.rst

```diff
@@ -0,0 +1,256 @@
+.. _protocols_serialization_server_app:
+
+Protocols serialization application
+###################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The protocols serialization server runs full Bluetooth LE and OpenThread stacks and it exposes selected functions from these stacks over the UART.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+Protocols serialization application performs server part of the OpenThread and Bluetooth LE API calls serialization between two devices
+that communicate with each other using the UART interface. The application uses nRF RPC library and CBOR encoding to serialize function calls.
```

Comment:
It's up to @annwoj but IMO some sentence like "You need a client device for testing this application. The reference client sample within |NCS| is (here put the link to ps_client)" might be useful.

Reply:
This information is probably best served in the Requirements section. I'd word it something like this:
```
To test the application, you need a client device.
The :ref:`nrf_rpc_protocols_serialization_client` sample is the reference implementation in the |NCS|.
```

Reply:
It has been referenced in Testing section: https://github.com/nrfconnect/sdk-nrf/pull/16597/files#diff-f98227582afec07af367ef80e56ca6042ac03b6b629d6cb50e7589c302d3bbdbR54-R57